partners/scientific-communities.md

```diff
@@ -9,26 +9,287 @@ JOSS <joss>
+review processes can leverage our existing peer review process, which is
+built on community-driven package standards. In partnership with these
+communities, we enhance our standard review process with a community-specific
+step that adds your specific requirements for package affiliation.
 
-## Software peer review support for scientific communities
-pyOpenSci is launching a prototype collaboration that supports an existing
-scientific community.
+Through this single end-to-end review process, a package can potentially:
+
+- Be accepted by pyOpenSci.
+- Undergo a fast-track review and get published with JOSS (if in scope).
+- Achieve affiliation with your domain-specific community.
+
+Here we discuss our process for
+supporting this community affiliation partnership.
+
+:::{figure-md} pyos-badge-home
+:class: myclass
+
+<img src="../images/peer-review-partners-process.png" alt="Add alt here">
+
+**Add caption**
+:::
+
+## What do we get when we become a pyOpenSci partner?
 
-### What pyOpenSci provides to existing communities
 In addition to [the core benefits that pyOpenSci peer review provides](/about/benefits),
-we also provide infrastructure and resources for:
-* Peer review of tools in the community's ecosystem
-* Adoption of community specific review standards which will be enforced in addition
- [to our existing review guidelines](editor-checklist-template)
-* Maintenance support to maintainers that go through our review.
-* Support for finding a new package maintainer in case the current one needs to step down.
-* A catalog of vetted tools and a community landing page.
-
-### Partnership benefits to the broader scientific community
-The benefit of communities partnering with us is:
-
-* We have a broad view across all scientific ecosystems which will help us to better identify overlap of package functions.
-* We also have broader reach to users of scientific software across the ecosystem.
-* Finally a consolidated peer review process will help enforce standardization of scientific Python packaging and best practices across the ecosystem.
+we will provide you with the following:
+
+**Visibility**
+
+1. We will create a community page in this peer review guide where you can define your guidelines for a package to become affiliated with your community. Check out the
+   [pangeo example](pangeo) as an example of what this looks like.
+2. A dedicated community page on the pyOpenSci website. Here, we will
+   list your community along with all affiliated packages that have
+   successfully undergone our review process. You can add descriptive text and graphics and branding to the page as you wish.
+
+**Packaging Support**
+
+Our vibrant community is filled with experts in the Python packaging space. A
+partnership with us means that you will always have support if you run into
+issues with packaging.
+
+**Coming in 2024**
+
+1. Access to metric dashboards for all accepted packages in the
```

Comment:
Numbered list does not quite make sense here unless you plan to add more?

Reply:
good point. i'll edit that locally. we also will be implementing a review bot.